api/integrations/slack.js

```diff
@@ -0,0 +1,73 @@
+var q = require('q');
+var Slack = require('slack-node');
+
+module.exports = function () {
+    global.app.post('/slack', function (req, res) {
+
+        var appId = req.body.appid;
+        var appkey = req.body.appkey;
+        var event_type = req.body.event_type;
+        var user = req.body.user;
+
+        console.log(appId, appkey, event_type);
+
+        var response = { appid: req.body.appid, status: "", user: user }
+
+        global.appService.isMasterKey(appId, appkey).then(function (isMasterKey) {
```

Comment:
dont check for masterKey, use isKeyValid instead , we want both master and client key to call this API.